.buildkite/scout_ci_config.yml

```diff
@@ -2,6 +2,7 @@
 plugins:
   enabled:
     - apm
+    - infra
```

Comment:
nit: could you move it to alphabetical order and also adjust `slo`? The list is growing and will be easier to search